modules/actions/workflows.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package actions
+
+import (
+	"bytes"
+	"io"
+	"strings"
+
+	"code.gitea.io/gitea/models/webhook"
+	"code.gitea.io/gitea/modules/git"
+	"code.gitea.io/gitea/modules/log"
+
+	"github.com/nektos/act/pkg/model"
+)
+
+func ListWorkflows(commit *git.Commit) (git.Entries, error) {
+	tree, err := commit.SubTree(".gitea/workflows")
+	if _, ok := err.(git.ErrNotExist); ok {
+		tree, err = commit.SubTree(".github/workflows")
```

Comment:
Does this mean, it will also list Workflows from github/workflows? I think, this is a bad Idea. Some Users may want to Mirror the Repo to GitHub to use the Windows or MacOS runner, which they don' have on their own Server. In this case, Gitea should not execute this.

Reply:
This means, it will check `.gitea/workflows` at first, if it does not exist, check `.github/workflows`. Since actions could be disabled per repo, I think we don't worry about the thing like you said?